rkt/prepare.go

```diff
@@ -77,6 +78,7 @@ func init() {
 	// argument. This is need to permit to correctly handle
 	// multiple "IMAGE -- imageargs ---"  options
 	cmdPrepare.Flags().SetInterspersed(false)
+	flagPorts = portList{}
```

Comment:
seems like there could be a more logical location for this (separate from this interspersed operation at least)